### ISSUE TITLE
Specifiying Relative file path problems with knitting

### ISSUE THREAD

**Issue by wvictor14** [STAT540-UBC/Discussion#138]
Hi @santina @farnushfarhadi @singha53 ,

I want to specify relative file paths (for reproducibility) when I'm writing code for that requires reading in an external data file. However, I've encountered an annoying problem when I try to knit my RMD file. Let me illustrate the problem with the following example:

I'm working with the Minfi package to read in DNAm data. So I want to read in this metadata file using the function **read.metharray.sheet(base, ...)** and store it in the variable **validation_sheet**. 

My current working directory is set to "C:/Users/Documents/Deconvolution/"
The external data sheet is stored two subdirectories down in "C:/Users/Documents/Deconvolution/Louie's data/validation raw data".

The following code runs without errors in console:

```
BASEDIR <- "../Deconvolution/Louie's data/validation raw data"
validation_sheet <- read.metharray.sheet(BASEDIR)
```
But when I try to knit I will get this error:

> Error in read.metharray.sheet(BASEDIR) : 'base' does not exist in Calls: <Anonymous> ... withVisible -> eval -> eval -> read.metharray.sheet Execution halted

I can knit this properly by changing BASEDIR to the following:

`BASEDIR <- "../Louie's data/validation raw data"

So basically every time I want to knit I have to change the file path, and then back if I want to run in console. I'm not sure why this problem is occurring or how to fix it.

**Comment by santina** [comment]
Hi @wvictor14 ,

Since your working directory is in `C:/Users/Documents/Deconvolution/`,  to get to your data, you'd do 
`BASEDIR <- "Louie's data/validation raw data"`  which basically would navigate from `Deconvolution` to your data, because `Louie's data` is in the same folder as your Rmarkdown or RScript. 

I'm not sure why ``BASEDIR <- "../Louie's data/validation raw data"` would work, unless your Rmarkdown or Rcript is in another folder that's inside `Deconvolution`.
